frontend_react/src/components/MapMarker/MapMarker.module.css

```diff
@@ -0,0 +1,33 @@
+.icon {
+  height: 30px;
+  width: 30px;
+  filter: drop-shadow(0px 2px 2px rgba(0, 0, 0, 0.4));
+  position: absolute;
+  transform: translate(-47%, -90%); /* Adjust for centering of icon */
```

Comment:
Is this the best way to do this? Its always risky to use percentages to locate elements. Could you think of a scenario where this would break?

Reply:
Updated to hard code pixel values instead.